var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -76,7 +76,7 @@ def install(self, spec, prefix):
             mkdirp(prefix.bin)
             for root, dirs, files in os.walk('.'):
                 for name in files:
-                    if name.endswith(".so") or name.endswith(".cc") \
+                    if name.endswith(".".format(dso_suffix)) or name.endswith(".cc") \
```

Comment:
There's nothing to format here, try `'.' + dso_suffix`

Reply:
my mistake, i wasn't paying attention to this change